x-pack/solutions/security/plugins/elastic_assistant/server/lib/attack_discovery/schedules/fields/field_map.ts

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FieldMap, alertFieldMap } from '@kbn/alerts-as-data-utils';
+import {
+  ALERT_ATTACK_DISCOVERY_ALERTS_CONTEXT_COUNT,
+  ALERT_ATTACK_DISCOVERY_ALERT_IDS,
+  ALERT_ATTACK_DISCOVERY_API_CONFIG,
+  ALERT_ATTACK_DISCOVERY_API_CONFIG_ACTION_TYPE_ID,
+  ALERT_ATTACK_DISCOVERY_API_CONFIG_CONNECTOR_ID,
+  ALERT_ATTACK_DISCOVERY_API_CONFIG_DEFAULT_SYSTEM_PROMPT_ID,
+  ALERT_ATTACK_DISCOVERY_API_CONFIG_MODEL,
+  ALERT_ATTACK_DISCOVERY_API_CONFIG_PROVIDER,
+  ALERT_ATTACK_DISCOVERY_DETAILS_MARKDOWN,
+  ALERT_ATTACK_DISCOVERY_ENTITY_SUMMARY_MARKDOWN,
+  ALERT_ATTACK_DISCOVERY_MITRE_ATTACK_TACTICS,
+  ALERT_ATTACK_DISCOVERY_REPLACEMENTS,
+  ALERT_ATTACK_DISCOVERY_SUMMARY_MARKDOWN,
+  ALERT_ATTACK_DISCOVERY_TITLE,
+  ALERT_ATTACK_DISCOVERY_USERS,
+  ALERT_ATTACK_DISCOVERY_USER_ID,
+  ALERT_ATTACK_DISCOVERY_USER_NAME,
+} from './field_names';
+
+export const attackDiscoveryAlertFieldMap: FieldMap = {
+  /**
+   * Default alert-as-data fields
+   */
+  ...alertFieldMap,
```

Comment:
Consider adding a mapping for `KIBANA_ALERT_RISK_SCORE`, because it does not appear to be defined by `alertFieldMap`. Both scheduled and ad hoc runs would `SUM()` the `kibana.alert.risk.score` fields of the alert IDs in the attack discovery to populate this field.

Reply:
done in https://github.com/elastic/kibana/pull/215184/commits/a1b967c4e67027be409c52a6f665f2db35fe405d